src/Uno.UI.Composition/Composition/ContainerVisual.skia.cs

```diff
@@ -1,10 +1,40 @@
 #nullable enable
 
-using SkiaSharp;
+using System.Collections.Generic;
+using System.Linq;
 
-namespace Windows.UI.Composition
+namespace Windows.UI.Composition;
+
+public partial class ContainerVisual : Visual
 {
-	public partial class ContainerVisual : Visual
+	private List<Visual> _childrenInRenderOrder = new List<Visual>();
```

Comment:
Since most visuals won't ever have children with non-zero Z indices, it may make sense to only allocate the list when it's actually needed.